pkg/util/util.go

```diff
@@ -66,3 +70,13 @@ func GetMountOptions(options []string) string {
 	}
 	return str
 }
+
+func MakeDir(pathname string) error {
```

Comment:
could you add a unit for this function @Sakuralbj

Reply:
ok